x16-smc.ino

```diff
@@ -264,8 +271,7 @@ void loop() {
 void DoPowerToggle() {
   LONGPRESS_START=0;		// Ensure longpress flag is 0
   if (bootloaderTimer > 0) {
-    bootloaderFlags |= 1;
-    startBootloader();
+    // Do nothing, is evaluated inside evaluateBootloaderKeypress()
```

Comment:
This if statement does nothing and is kind of obsolete. I think it should be removed.

Reply:
It does serve a purpose as is, which is to "disable the power button" while bootloaderTimer is active (as it then prevents the following "else if" blocks to execute). You do not want the SMC to actually power off the power supply when doing the button combination.

I can improve the comment to highlight this effect. Or, I can replace it with a "return" if that makes it cleaner. Or, do you see any other solution, as you suggest to remove it?

The same applies to reset and NMI. I opted for using the same logic all places.

Reply:
It's just a bit strange to have an if statement that does nothing.

Maybe it's best to keep it. It could be rewritten like this though:

```
if (bootloaderTimer == 0) {
  if (SYSTEM_POWERED == 0) {                  // If Off, turn on
    PowerOnSeq();
  }
  else {                                      // If On, turn off
    PowerOffSeq();
  }
}
```